app/queries/items_by_storage_collection.rb

```diff
@@ -0,0 +1,31 @@
+module Query
+  class ItemsByStorageCollection
+    attr_reader :organization
+    attr_reader :filter_params
+
+    def initialize(organization:, filter_params:)
+      @organization = organization
+      @filter_params = filter_params
+    end
+
+    # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
I'd like to settle on the convention of using `MyQueryObject#call` for when we want to use a query object, and I'd rather remove memoization than rename the returned variable to be `@call`.  Or we can disable the cop here.  So many choices that are better than _"rename the returned variable to be `@call`"_!